src/main/scala/org/broadinstitute/clio/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -1,5 +1,6 @@
 package org.broadinstitute.clio.dataaccess
 
+import java.lang.{Boolean => JBoolean, Double => JDouble, Float => JFloat, Integer => JInteger, Long => JLong}
 import java.time.OffsetDateTime
```

Comment:
It was available in the mapper, but wasn't being passed. I was just thinking ahead, as I didn't want future devs trying to leak in [Joda-Time](http://www.joda.org/joda-time/), or even java.util.Date.